src/test.ts

```diff
@@ -26,7 +26,7 @@ function compareToBaselines() {
 
 function compileGeneratedFile(file: string) {
     try {
-        child_process.execSync(`node ${tscPath} --strict --lib es5 --noEmit ${path.join(outputFolder, file)}`);
+        child_process.execSync(`node ${tscPath} --strict --lib es5 --skipLibCheck --noEmit ${path.join(outputFolder, file)}`);
```

Comment:
why? we want to make sure we are generating valid declaration file.. that is the whole point of the test.

Reply:
This breaks `@types/jsdom` which depends on DOM types. I think we are still testing the input library file, this just turns off checks for JavaScript types.

Reply:
Then Add ‘—types’ to exclude all @types packages.

Reply:
Good idea, but how can I pass empty string to `--types`? `--types []`, `--types ""`, or `--types "''"` don't work.

Reply:
just `--types`.. 

`--strict --lib es5 --types --noEmit`